### PR TITLE
Add example codesnippet for using ArgsTable with stories

### PR DESCRIPTION
**Pull request by mattdarveniza**
Issue:

It's unclear from the docs how to use `ArgsTable` in mdx with stories, which is currently the only way to customise ArgTypes in mdx. As seen in #11979, the only hint this is possible is at https://storybook.js.org/docs/react/writing-docs/doc-blocks#customizing where it references using a `story` attribute.

## What I did

* Added a second `mdx` example of showing an ArgsTable for a specific story.
* Renamed first example to distinguish from second example.

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@mattdarveniza thank you for the time and effort you put into this, let me take a look at it and see if everything is alright, or we can fine tune the documentation a bit and improve it.

Sounds good?

**Comment by mattdarveniza** [comment]
Yep open to suggestions or tweaks, this was just the first thing that came to mind 👍

**Comment by darleendenno** [comment]
@jonniebigodes does this look good to merge?

**Comment by jonniebigodes** [comment]
@mattdarveniza sorry for not addressing this sooner and truly sorry for not addressing it sooner 🙈. If you wouldn't mind fixing the conflicts I'm more than glad to merge this and improve the docs. @darleendenno thanks for putting this on my "radar" (pardon the bad pun), this is good to go on my end once the conflicts are dealt with.

File: docs/snippets/common/component-story-mdx-argstable-block-for-story.mdx.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+```md
+<!--- MyComponent.stories.mdx -->
+
+import { ArgsTable } from '@storybook/addon-docs/blocks';
+import { MyComponent } from './MyComponent';
```
Comment (jonniebigodes) [inline docs/snippets/common/component-story-mdx-argstable-block-for-story.mdx.mdx:5]: @mattdarveniza one quick update on this:
```js
import { ArgsTable } from '@storybook/addon-docs/blocks'
```
This should be:
```js
import { ArgsTable } from '@storybook/addon-docs/';
```

**Review by shilman — COMMENTED** [review]
LGTM! thansk @mattdarveniza

**Comment by mattdarveniza** [comment]
Thanks @shilman and @jonniebigodes! No stress on the delay ✌️

**Pull request merged**